{{cookiecutter.project_slug}}/Makefile

```diff
@@ -139,21 +158,37 @@ upload-all: upload-code upload-data upload-notebooks  ### Upload code, data, and
 .PHONY: clean-all
 clean-all: clean-code clean-data clean-config clean-notebooks  ### Delete code, data, config and notebooks directories from the platform storage
 
+##### Google Cloud Integration #####
+
+.PHONY: gcloud-check-auth
+gcloud-check-auth:  ### Check if the file containing Google Cloud service account key exists
+	@echo Using variable: GCP_SECRET_FILE=${GCP_SECRET_FILE}
+	@test ${GCP_AUTH_EXISTS} \
+		&& echo "Google Cloud will be authenticated via service account key file: '${GCP_SECRET_PATH_LOCAL}'" \
+		|| { echo -e "ERROR: Not found Google Cloud service account key file: '${GCP_SECRET_PATH_LOCAL}'.\nPlease put the key file named '$${GCP_SECRET_FILE}' to '${CONFIG_DIR}/'"; false; }
+
 ##### JOBS #####
 
 .PHONY: develop
-develop: upload-code upload-notebooks  ### Run a development job
+develop: upload-code upload-config upload-notebooks  ### Run a development job
 	$(NEURO) run \
 		--name $(DEVELOP_JOB) \
 		--preset $(PRESET) \
 		--detach \
 		--volume $(DATA_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(DATA_DIR):ro \
 		--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):rw \
+		--volume $(PROJECT_PATH_STORAGE)/$(CONFIG_DIR):$(PROJECT_PATH_ENV)/$(CONFIG_DIR):ro \
 		--volume $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
+		${OPTION_GCP_CREDENTIALS} \
 		--env EXPOSE_SSH=yes \
 		--env JOB_LIFETIME=1d \
 		$(CUSTOM_ENV_NAME) \
 		"sleep infinity"
+		@echo -e "[+] Job ${DEVELOP_JOB} successfully started!\n[!] PLEASE DON'T FORGET TO KILL it via 'make kill-develop'\n"
```

Comment:
Let me know if you don't like the formatting. In future, I would also print some shortcuts on what to do next: `make connect-develop`, `make logs-develop`, etc.